stor/obs.py

```diff
@@ -1,7 +1,12 @@
-import cStringIO
 import posixpath
+import sys
 
 from cached_property import cached_property
+from six import BytesIO
+try:
+    from cStringIO import StringIO
+except ImportError:  # pragma: no cover
+    from six import StringIO
```

Comment:
Ah I was unaware of its issues handling certain strings. @jtratner any objections to just changing it all to use io.StringIO and io.BytesIO?